css/css-flexbox/abspos/flex-abspos-staticpos-align-content-008.html

```diff
@@ -39,9 +39,13 @@
       height: 6px;
       width: 8px;
       /* This "align-self" only gets a chance to take effect when our container
-         has "align-content: stretch". In that case, it helps us verify that
-         the container's "align-content: stretch" is actually taking effect
-         and stretching the flex line (and giving us space to center in). */
+         has "align-content: stretch" and its available size is non-negative.
+         In that case, align-self helps us verify that the container's
+         "align-content: stretch" is actually taking effect and stretching the
+         flex line (and giving us space to center in). In the case of a negative
+         available size, "align-content: stretch" should not give us any space
+         to center in and so we should see the fallback behavior to
+         "align-content: flex-start". */
```

Comment:
For this test, it looks like this commit is updating the comment but not the test expectation.

This one needs a test expectation update just like the others.  (The `small` `normal`/`stretch` cases should get `data-offset-x="-2"` to match `flex-start`, I think?)